core/test/golden/json/Address0

```diff
@@ -0,0 +1 @@
+"2RhQhCGqYPDpFgTsnBTbnvPvCwpqAkjwLqQkWpkqXbLRmNxd4xNd262nGsr8JiynyKRUeMLSJ9Ntho9i76uvBTrVXdJJG5yiNLb8frmUe5qX7E"
```

Comment:
Was this JSON encoded? Same question for the other example addresses.

Reply:
Yeah good question - that should've raised eyebrows for me but I was too focused on finishing the job to notice.

It seems, though, like that's just the JSON format of `Address` for us: [toJSON](https://github.com/input-output-hk/cardano-sl/blob/6a73bb5d0/core/src/Pos/Core/Common/Address.hs#L154) calls [toObjectKey](https://github.com/input-output-hk/cardano-sl/blob/6a73bb5d0/core/src/Pos/Core/Common/Address.hs#L148) which uses [addressF](https://github.com/input-output-hk/cardano-sl/blob/6a73bb5d0/core/src/Pos/Core/Common/Address.hs#L202) to call [build](https://github.com/input-output-hk/cardano-sl/blob/6a73bb5d0/core/src/Pos/Core/Common/Address.hs#L198).

Let me know what you think, since you've worked in the Json parts of the codebase more.

Reply:
Yep this is the right encoding, just double checked in GHCi. Definitely looks different to the usual key/value map style encoding.

Reply:
Cool, thanks for confirming.